src/main/java/com/gmail/goosius/siegewar/events/SiegeEndEvent.java

```diff
@@ -89,4 +92,8 @@ public String getBesiegedTownName() {
 	public String getDefenderName() {
 		return defenderName;
 	}
+
```

Comment:
Ought to have a @Nullable annotation, because I believe this can be null.

Reply:
I think you put the @Nullable on the wrong method, now that I've looked at the code longer I don't believe getNation() can be null.